Frontend challenge/candidates

rafapaulin opened the pull request:
Layout desktop/mobile da página de acompanhamento de pedidos.

na sequencia: página de login
à seguir: simulação de dados colhidos de banco de dados

rafapaulin committed:
Initial setup, dependencies, templates and libraries

rafapaulin committed:
base template

rafapaulin committed:
order controller, order layout, cards

- Adicionado Controller para os pedidos
  - Adicionada View para orders/show
- Adicionado layout para card details

rafapaulin committed:
Orders, card status

- Adicionado card com status do pedido
  - Timeline colapsável

rafapaulin committed:
Orders, products

- Adicionado terceiro card, detalhando produtos
- Adicionada timeline interativa horizontal
- Ajustado primeiros bugs mobile (precisa refinar)

rafapaulin committed:
- Projeto movido para pasta com nome do candidato

nandomoreirame commented:
@rafapaulin, consegue resolver o conflito pra gente? :smiley:

nandomoreirame commented:
Na verdade faz assim, adiciona as extensões que você adicionou no .gitignore do projeto em seu .gitignore global :)

rafapaulin committed:
Ajustando conflitos

rafapaulin committed:
Ajustando conflitos

rafapaulin commented:
Conflitos ajustados. Peço desculpas por nao ter corrigido na primeira vez. Acabei passando batido hehe

rafapaulin committed:
Login form

- Adicionado Formulário de login
  - Validação para campo de número de pedidos (somente números, não vazio)
  - Validação para campo de CPF/CNPJ (11 ou 14 dígitos, somente númentos, não vazio)

rafapaulin committed:
- Login "fake-funcionando"
  - Ao efetuar o login, redireciona para página de acompanhamento
    utilizando o número do pedido passado

rafapaulin committed:
Orders

- Página de acompanhamento funciona "dinamicamente" (ver a
  controller OrderController)

rafapaulin commented:
Pronto, finalizado!

Instalando o laravel na pasta, executando a instalação das dependências do composer e to npm, basta executar "php artisan serve" na pasta do projeto, e acessar /login

o login valida o form e "funciona", redirecionando para a página de acompanhamento de pedidos, que é preenchida "dinamicamente".

Fico no aguardo do feedback, e peço desculpas pela demora em entregar o teste... estava finalizando um freela hehe.

emails para contato: rafapaulin@gmail.com ou rafael@repdesign.com.br

Obrigado pela oportunidade!

nandomoreirame commented:
:+1:

rafapaulin commented:
algum feedback atm?
